matching/src/main/scala/org/kframework/backend/llvm/matching/Heuristics.scala

```diff
@@ -149,6 +149,31 @@ object AHeuristic extends Heuristic {
   }
 }
 
+object LHeuristic extends Heuristic {
+  val needsMatrix: Boolean = true
+
+  def computeScoreForKey(c: AbstractColumn, key: Option[Pattern[Option[Occurrence]]]): Double = {
+    var result = 0.0
+    val matrixColumn = c.asInstanceOf[MatrixColumn]
+    val matrix = matrixColumn.matrix
+    val colIx = matrixColumn.colIx
+    matrix.columns.updated(0, matrix.columns(colIx)).updated(colIx, matrix.columns(0))
```

Comment:
not necessary. we don't actually implement "swaps" in the paper as actual swaps.